lib/requests/create-account-request.js

```diff
@@ -26,12 +26,16 @@ class CreateAccountRequest extends AuthRequest {
    * @param [options.response] {HttpResponse}
    * @param [options.returnToUrl] {string} If present, redirect the agent to
    *   this url on successful account creation
+   * @param [options.enforceToc] {boolean} Whether or not to enforce the service provider's T&C
+   * @param [options.tocUri] {string} URI to the service provider's T&C
+   * @param [options.acceptToc] {boolean} Whether or not user has accepted T&C
    */
   constructor (options) {
     super(options)
 
     this.username = options.username
     this.userAccount = options.userAccount
+    this.acceptToc = options.acceptToc === 'true'
```

Comment:
should that parsing be this constructor's job?

Reply:
Probably `fromParams` (even though the whole `fromParams` pattern all over is a disease)